File: 2022.01.29/javascript/2920.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n')[0].split(' ').map(Number);
let as = des = true;
let result = "mixed";
for (let i = 1 ; i<input.length ; i++){
    if (!des && !as) break;
    if (input[i]>input[i-1]) des = false;
    if (input[i]<input[i-1]) as = false;
}
if (as) result = "ascending";
if (des) result = "descending"
console.log(result);

File: 2022.01.29/python/2920.py
nums = list(map(int,input().split()))
as_bool = des_bool = True
result = "mixed"
for i in range(1,len(nums)):
    if not (as_bool or des_bool) : break
    if nums[i]>nums[i-1]: des_bool = False
    if nums[i]<nums[i-1]: as_bool = False
if as_bool : result = "ascending"
if des_bool : result = "descending"
print(result)